lib/rules/keyframe-selector-notation/index.js

```diff
@@ -73,6 +74,10 @@ const rule = (primary, _, context) => {
 
 			atRuleKeyframes.walkRules((keyframeRule) => {
 				transformSelector(result, keyframeRule, (selectors) => {
+					if (isNamedTimelineRange(selectors)) {
+						return;
+					}
+
```

Comment:
[suggest] How about the following refactoring to avoid extra selector node iteration within `isNamedTimelineRange()`? When a `walk` callback returns `false`, the iteration stops.

```js
transformSelector(result, keyframeRule, (selectors) => {
	let first = true;

	selectors.walkTags((selectorTag) => {
		if (first && NAMED_TIMELINE_RABGE_SELECTORS.has(selectorTag.value)) {
			return false; // ignore named timeline range
		}

		first = false;

		checkSelector(...);
	});
});
```

Reply:
Thanks. Your suggestion is simpler and better, so I fixed it. https://github.com/stylelint/stylelint/pull/6605/commits/80a8351c20c1c14d6dfebd7dfc2580296d2e567a